dev/.rat-excludes

```diff
@@ -98,3 +98,4 @@ LZ4BlockInputStream.java
 spark-deps-.*
 .*csv
 .*tsv
+org.apache.spark.scheduler.ExternalClusterManager
```

Comment:
Rat throws an error for the files that do not have apache license. Hence excluding this file.